README.md

```diff
@@ -1,34 +1,148 @@
-# Hello World
-Hello World is a project about Introduction to Microverse
-# Built With
-it is built with Html & CSS
-# Key Features
-Html
-css
-# Getting Started
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140" height="auto" />
+  <br/>
+
+  <h3><b>README.md</b></h3>
+
+</div>
```

Comment:
- [ ] You essentially will need to remove this section along with the Microverse logo because I guess you may not have a logo at this point.  If you have your logo, you can use it but I would advise you to remove it. 🙏